demos/common/python/openvino/model_zoo/model_api/adapters/openvino_adapter.py

```diff
@@ -40,7 +40,7 @@ def create_core():
 
 class OpenvinoAdapter(ModelAdapter):
     """
-    Class that allows working with Inference Engine model, its input and output blobs
+    Works with OpenVINO model, its inputs and outputs
```

Comment:
```suggestion
    Works with OpenVINO model
```
Maybe it is worth to remove this phrase, every adapter works with inputs and outputs